### PR TITLE
Create README.md for ratadns docker

### PR DESCRIPTION
**Pull request by camilog**

### CODE REVIEW

File: ratadns/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# RaTA DNS - Docker images
+
+## RaTA DNS
+
+Real time analytics for DNS servers, from the packets capture to the web visualization of the data. Project available [here](https://www.github.com/niclabs/ratadns).
```
Comment (fmontoto) [inline ratadns/README.md]: s/packets/packet

File: ratadns/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Dockerfile to create container that runs a clean instance of Fievel.
+
+### gopher
+
+Dockerfile to create container that runs an instance of Gopher, exposing port 8000 and using gunicorn as WSGI application container.
+
+### nginx
+
+Dockerfile that runs an nginx server necessary to run web visualization Remy.
```
Comment (fmontoto) [inline ratadns/README.md]: This could be the nginx docker explanation. Why is it different than a vanilla nginx?

Comment (camilog) [reply ratadns/README.md]: I was confused with the name, actually is a Dockerfile for Remy using nginx image as base.

**Comment by fmontoto** [comment]
LGTM

**Comment by Zeeerooo** [comment]
LGTM

**Comment by sebablasko** [comment]
lgtm

**Pull request merged**